Add `gpu-encoder-slots` (#208)
anbox-cloud/discourse-docs: howto/cluster/configure-nodes.md

<a name="configure-gpu-slots"></a>
## Configure GPU slots and GPU encoder slots

GPU slots are used to share GPUs amongst containers. See [About GPU support](https://discourse.ubuntu.com/t/gpu-support/17768) and [GPU slots](https://discourse.ubuntu.com/t/about-capacity-planning/28717#gpu-slots) for more information.

Each GPU-equipped cluster node is configured with a number of GPU slots and a number of GPU encoder slots. See [Node-specific configuration](https://discourse.ubuntu.com/t/ams-configuration/20872#node-specific) for the default values that are used. Nodes without GPU are configured with 0 GPU slots and 0 GPU encoder slots.

Use the following commands to change the number of GPU slots and GPU encoder slots for a node:

    amc node set <node> gpu-slots <number>
    amc node set <node> gpu-encoder-slots <number>

Replace `<node>` with the node name (for example, `lxd0`), and `<number>` with the number of slots.

Use the following commands to set the number of GPU slots and GPU encoder slots for each GPU of a node:

    amc node set <node>.gpus.<n>.slots <number>
    amc node set <node>.gpus.<n>.encoder-slots <number>

> Replace `<node>` with the node name (for example, `lxd0`), `<n>` with the GPU number, and `<number>` with the number of slots.

Author: Yeah, I saw the guidance about not using it, I just forgot. I personally like it and have been using it all my life as a technical author but I would usually try and stick to the existing style. I will remove it.

    amc node set <node> memory-allocation-rate <value>

<a name="configure-gpu-slots"></a>
## Configure GPU slots and GPU encoder slots

GPU slots are used to share GPUs amongst containers. See [About GPU support](https://discourse.ubuntu.com/t/gpu-support/17768) and [GPU slots](https://discourse.ubuntu.com/t/about-capacity-planning/28717#gpu-slots) for more information.

Each GPU-equipped cluster node is configured with a number of GPU slots and a number of GPU encoder slots. See [Node-specific configuration](https://discourse.ubuntu.com/t/ams-configuration/20872#node-specific) for the default values that are used. Nodes without GPU are configured with 0 GPU slots and 0 GPU encoder slots.

Use the following commands to change the number of GPU slots and GPU encoder slots for a node:

    amc node set <node> gpu-slots <number>
    amc node set <node> gpu-encoder-slots <number>

> where,
> `<node>` is the node name (For example, `lxd0`),
> `<number>` is the number of slots.

Reviewer: I don't think this will display nicely in Markdown. I would use either a sentence:
```suggestion
Replace `<node>` with the node name (for example, `lxd0`) and `<number>` with the number of slots.
```
Or a list:
```suggestion
Replace the following variables:

- `<node>` is the node name (for example, `lxd0`).
- `<number>` is the number of slots.
```

And the same below. :)
